dapp/src/components/TransactionListener.js

```diff
@@ -272,7 +271,9 @@ const TransactionListener = ({ rpcProvider }) => {
         // TODO handle a 404 here. We need to retry. A refresh is required to
         // get this to retry at the moment.
         try {
-          safeData = await connector.sdk.txs.getBySafeTxHash(t.hash)
+          // TODO
+          console.log(connector)
```

Comment:
same thing, do you need console.log in prod?

Reply:
Correct, this actually isn't done yet. I need to adjust this equivalent from web3 react. Left a TODO, revisiting it, good call out though.

Reply:
@smitch88 does it mean that it's not ready to be merged?

Reply:
This is just an issue with transaction display for gnosis specifically. I think we could merge and unblock most of things while fixing this as a follow up.